File: 卡码/代码随想录/1/03-哈希表/Python/01-有效字母异位词/LeetCode_0242.py
#242. 有效的字母异位词
class Solution:
	def isAnagram(self, s: str, t: str) -> bool:
			hash=[0 for i in range(26)]
			for ch in s:
				hash[ord(ch)-ord('a')]+=1
			for ch in t:
				hash[ord(ch)-ord('a')]-=1
			for num in hash:
				if num!=0:
					return False
			return True

File: 卡码/代码随想录/1/03-哈希表/Go/01-有效字母异位词/LeetCode_0242.go
//242. 有效的字母异位词
package main

func isAnagram(s string, t string) bool {
	hash:=make([]int,26)
	for _,ch:=range s {
		hash[ch-'a']++
	}
	for _,ch:=range t {
		hash[ch-'a']--
	}
	for _,num:=range hash {
		if num!=0 {
			return false
		}
	}
	return true
}
